### DISCUSSION TITLE
How to get iTerm2 shell integration to work?

### DISCUSSION THREAD

**Discussion by benkeil** [starship/starship#3818] [Q&A]
I use fish and everything works fine without starship. How to configure it to run with Starship?

**Comment by imrekoszo** [comment]
Having the same problem. Have you found a solution yet?

**Reply by benkeil** [reply]
No success

**Reply by imrekoszo** [reply]
:( too bad, thank you.

There certainly has been some related activity in the past: https://github.com/gnachman/iTerm2/blob/master/Resources/shell_integration/iterm2_shell_integration.fish#L82

**Reply by imrekoszo** [reply]
I suppose this could be worth an issue with starship and perhaps iterm2 too. When I have time to set up a minimal repro I'll submit them.

**Comment by luckman212** [comment]
related? https://github.com/starship/starship/issues/4642
